File: eval/fixtures/legacy-directory-backed-wcf-commerce/src/Legacy/Billing/BillingRefundService.cs
namespace Acme.Legacy.Billing;

public class BillingRefundService : IBillingRefundContract
{
  private readonly BillingRefundManager _manager = new BillingRefundManager();

  public string AuditRefund(string refundId)
  {
    return _manager.RebuildRefundAudit(refundId);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/billing/BillingRefundService.java
package com.acme.legacy.billing;
import javax.jws.WebMethod;
import javax.jws.WebService;

@WebService(serviceName = "BillingRefundService")
public class BillingRefundService {
  private final BillingRefundManager billingRefundManager = new BillingRefundManager();

  @WebMethod(operationName = "AuditRefund")
  public RefundAuditResult auditRefund(String refundId) {
    return billingRefundManager.rebuildRefundAudit(refundId);
  }
}
